features/FEATURE_UVISOR/includes/uvisor/api/inc/uvisor-lib.h

```diff
@@ -17,6 +17,8 @@
 #ifndef __UVISOR_API_UVISOR_LIB_H__
 #define __UVISOR_API_UVISOR_LIB_H__
 
+#include "includes/uvisor-lib/uvisor-lib.h"
```

Comment:
If this is needed, it's probably because some other file isn't including `uvisor-lib/uvisor-lib.h` correctly. Can you get away with not having this? What breaks? Can you include `uvisor-lib/uvisor-lib.h` from there instead?